packages/datagrid/src/I18N.js

```diff
@@ -0,0 +1,8 @@
+import { createInstance } from 'i18next';
+
+const I18N_DOMAIN_DATAGRID = 'tui-datagrid';
```

Comment:
Can you follow the same files as the other modules please ? `constants.js` and `translate.js`